### DISCUSSION TITLE
How to change the default workdir?

### DISCUSSION THREAD

**Discussion by aryan9600** [lima-vm/lima#586] [Q&A] [answered]
I removed the home directory mount, and it looks like the default workdir is the home directory. So whenever I try to use the vm, I get this:
```
❯ limactl shell default
bash: line 1: cd: /Users/sanskarjaiswal: No such file or directory
bash: line 1: cd: /Users/sanskarjaiswal: No such file or directory
sanskarjaiswal@lima-default:~$
```

**Comment by jandubois** [comment, marked as answer]
These are just warnings/noise, so you could ignore them. Alternatively you can specify the work directory explicitly:

```console
$ limactl shell default --workdir /
```

If you are using the `lima` wrapper, you could add the `--workdir /` argument in there.

**Reply by robrecord** [reply]
This doesn't work for me. I get `/bin/bash: --: invalid option`

Also is there a way to stop these "warnings" from being printed (or the `cd` commands from being invoked)? It's quite disconcerting for new users.

**Reply by robrecord** [reply]
OK it's actually:

```
limactl shell [flags] [instance] [command]
```

and not the other way round.

So:

```
limactl shell --workdir / default
```

which makes sense because otherwise it would interfere with any commands being sent. This should probably be updated in the help text.

**Reply by jandubois** [reply]
> This should probably be updated in the help text.

Yes, it should #1444
